e2e/test/scenarios/visualizations-charts/bar_chart.cy.spec.js

```diff
@@ -310,7 +310,7 @@ describe("scenarios > visualizations > bar chart", () => {
       cy.get("g.axis.yr").should("not.exist");
     });
 
-    it("should not split the y-axis on native queries with two numeric columns", () => {
+    it("should split the y-axis on native queries with two numeric columns", () => {
```

Comment:
It should still behave the old way, why did you change this spec to verify incorrect behavior?